components/SideMenu.js

```diff
@@ -77,11 +75,11 @@ class SideMenu extends Component {
 	}
 }
 
-SideMenu.propTypes = {
+SideMenu.propTypes = { // Do we need this?
```

Comment:
i would say we can leave this. it doesnt do anything but as long as i know its just a typecheck